java/memory/src/main/java/org/apache/arrow/memory/rounding/DefaultRoundingPolicy.java

```diff
@@ -31,19 +28,18 @@
 
   public final long chunkSize;
 
+  /**
+   * default chunk size from Netty.
+   */
+  private static final long DEFAULT_CHUNK_SIZE = 16777216L;
```

Comment:
Is it possible that this value changes from Netty code base?

Reply:
It is possible that it changes. It was a bit involved to copy the Netty code to get chunk size into the arrow codebase so I left it at a likely value for x86-64 machines. I am happy to replace this with the Netty value or have chunk size taken from the actual allocator directly if you thikn that is better than a single static value?

Reply:
Maybe it is beneficial to add above discussion to the JavaDoc

Reply:
done